adagrams/game.py

```diff
@@ -1,11 +1,115 @@
+import random
+
+def create_letter_list():
+    #put the helper function here
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    letter_pool = []
+    for letter in LETTER_POOL:
+        for i in range(LETTER_POOL[letter]):
+            letter_pool.append(letter)
+    return letter_pool
+
 def draw_letters():
-    pass
+    #convert LETTER_POOL dict to list--helper function?
+    letter_pool = create_letter_list()
+    hand = []
+    for i in range(10):
+        rand_index = random.randint(0, len(letter_pool) - 1)
+        hand.append(letter_pool.pop(rand_index))
+    return hand
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word = word.upper()
+    hand_copy = letter_bank[:]
+    # print(hand_copy)
+    for letter in word:
+        if letter not in hand_copy:
+            hand_copy = letter_bank[:]
+            return False
+        hand_copy.remove(letter)
+        # print(hand_copy)
+    return True
```

Comment:
This works well! Smart to copy the `letter_bank` to avoid mutating it. However, line 54 is unneeded.